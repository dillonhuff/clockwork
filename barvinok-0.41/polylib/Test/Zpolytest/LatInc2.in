#inculsion between the  affine lattices
M 3
3 3
2 0 0 
0 5 0
0 0 1
3 3
2 0 4 
5 5 15
0 0 1
3 3
4 0 8
15 15 15
0 0 1
D 0
F 4

2 = 1
2 \in 3