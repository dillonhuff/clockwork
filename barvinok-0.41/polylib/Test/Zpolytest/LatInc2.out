 2 in 1 : 1
 1 in 3 : 1
 1 in 2 : 1
